src/expressApp.ts

```diff
@@ -41,4 +55,19 @@ export function setupExpressApp(app: express.Application) {
       }
     }
   });
+
+  app.get("/robots", async (req, res) => {
+    // to access this endpoint, you need to provide token auth
+    // let email = undefined;
+    // const token = req.body["id_token"];
+    // if (googleAuthAvailable && typeof token === "string") {
+    //   const payload = await getAuthPayload(token);
+    //   if (payload?.email_verified && payload.email)
+    //     email = payload.email;
+    // }
+
```

Comment:
Do we think we need authentication for this endpoint?

Reply:
This is a good question to ask @joydeep-b: is [this information](https://github.com/ut-amrl/robofleet_server/pull/4/files#diff-ed28f093f74966cde0e38aaa50678450R3-R9), for any robot ever, sensitive?
If so, the overview page in webviz, which is designed to be public, needs to fail gracefully without it. 

_On adding auth here:_
Currently, auth is modeled in terms of every protected resource being a ROS topic (and I really think Robofleet should be dealing only in ROS topics wherever possible), so I think there are two cases:
1. the information is not sensitive: you could just not add auth, but to be safe you could check for read permission on `/status`, since "The server allows anyone to receive messages on any topic that ends in /status" (readme)
2. you could invent some nonexistent topic, such as `/robotStaticData`, or `/[robot name]/robotStaticData`, depending on whether you want static data permission to be granted on a per-robot level

We don't support _specifying_ per-topic authorization yet, and therefore don't fully check the topic passed into [`authorize()`](https://github.com/ut-amrl/robofleet_server/blob/9ccdbfb0f24a3586ffa63af04faa8b37a1cb36f8/src/access.ts#L20-L66), but that can be implemented.

Reply:
The name, internal IP, lastLocation, lastStatus, and lastUpdated fields are not sensitive, they can be viewable by anyone.